test/instance.js

```diff
@@ -38,6 +51,27 @@ function init(ndx) {
   return instance;
 }
 
+function initFull(ndx) {
+  assertJsDOM();
+  clear();
+
+  if (BROWSER) {
+    throw new Error('Full mode can only be run in non-browser mode');
```

Comment:
Given that this is in the `test/` directory, in what situation do we expect `BROWSER` to be set to true?

Reply:
See previous comment.